debugger_implementation.tex

```diff
@@ -392,6 +392,81 @@ \subsection{Using Program Buffer} \label{deb:mrprogbuf}
 \end{tabulary}
 \medskip
 
+\section{Triggers}
+
+A debugger would use hardware triggers to halt a hart when a certain event
+occurs. Below are some examples. There's no requirement on the number and
```

Comment:
Below are some examples, but as there is no requirement on the number of features of the triggers implemented by a hart, these examples may not be applicable to all implementations.

Reply:
Done